pkg/runtime/runtime.go

```diff
@@ -807,6 +812,12 @@ func (a *DaprRuntime) subscribeTopic(parentCtx context.Context, name string, top
 			}
 		})
 		if err != nil && err != context.Canceled {
+			if errors.Is(err, ErrRetriable) {
```

Comment:
This code seems to be executed after the retries were exhausted. The change should take place inside the policy callback function (lines 805 to 812 in this version.